<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEditExecutionTask</class>
 <widget class="QDialog" name="DlgEditExecutionTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre</string>
       </property>
       <property name="buddy">
        <cstring>txtNombre</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtNombre"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>TIpo de Tarea</string>
       </property>
       <property name="buddy">
        <cstring>cboTaskType</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cboTaskType"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tarea</string>
       </property>
       <property name="buddy">
        <cstring>cboTarea</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cboTarea"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Proveedor</string>
       </property>
       <property name="buddy">
        <cstring>cboProveedor</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cboProveedor"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Cantidad</string>
       </property>
       <property name="buddy">
        <cstring>txtCantidad</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="txtCantidad"/>
       </item>
       <item>
        <widget class="QLabel" name="lblUnidadMedida">
         <property name="text">
          <string>Unidad de medida</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="PorcetnajeRealizado">
       <property name="text">
        <string>Porcentaje realizado</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="spinPct"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Fecha Estimada Inicio</string>
       </property>
       <property name="buddy">
        <cstring>dateFechaEstimadaInicio</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDateEdit" name="dateFechaEstimadaInicio"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Fecha Estimada Finalización</string>
       </property>
       <property name="buddy">
        <cstring>dateFechaEstimadaFinalizacion</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDateEdit" name="dateFechaEstimadaFinalizacion"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Fecha Real Inicio</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDateEdit" name="dateFechaRealInicio"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Fecha Real Finalización</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDateEdit" name="dateFecharealFinalizacion"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Relaciones</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgEditExecutionTask</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgEditExecutionTask</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
